http/analysis-service/server.py

```diff
@@ -126,15 +128,23 @@ def update_job(job):
     update = table.update().where(table.c.id == job['id']).values(job)
     return db['conn'].execute(update)
 
-def get_jobs(owner=None):
+def get_jobs(owner=None, is_cluster=None):
     db = get_db()
     table = db['metadata'].tables['scheduled_jobs']
+
+    if is_cluster is None:
+        condition = True
+    elif is_cluster:
+        condition = (table.c.num_workers != None)
```

Comment:
http://stackoverflow.com/questions/5602918/postgresql-select-null-values-in-sqlalchemy